lib/versioning/conan/index.ts

```diff
@@ -0,0 +1,513 @@
+import * as semver from 'semver';
+import { parseRange } from 'semver-utils';
+import { logger } from '../../logger';
+import { api as looseAPI } from '../loose';
+import type { NewValueConfig, VersioningApi } from '../types';
+import {
+  cleanVersion,
+  containsOperators,
+  fixParsedRange,
+  getOptions,
+  matchesWithOptions,
+} from './common';
+
+export const id = 'conan';
+export const displayName = 'conan';
+export const urls = [
+  'https://semver.org/',
+  'https://github.com/podhmo/python-node-semver',
+  'https://github.com/podhmo/python-node-semver/tree/master/examples',
+  'https://docs.conan.io/en/latest/versioning/version_ranges.html#version-ranges',
+];
+export const supportsRanges = true;
+export const supportedRangeStrategies = ['auto', 'bump', 'widen', 'replace'];
+
+const MIN = 1;
+const MAX = -1;
+
+const makeVersion = (
```

Comment:
use full function declaration for all those functions and move this to `./common.ts`